### PR TITLE
docs: add champions concept

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR introduces the champion concept.

**Related issue(s)**
Related discussion: https://github.com/asyncapi/modelina/discussions/487

### CODE REVIEW

File: CODEOWNERS
Comment (jonaslagoni) [inline CODEOWNERS:10]: 🤔

File: docs/champions.md
Comment (magicmatatjahu) [inline docs/champions.md:26]:
```suggestion
## Becoming a champion
```

File: README.md
Comment (jonaslagoni) [reply README.md:124]: I honestly love we point out the focus on documentation, cause this is not explicit in libraries that they focus on it 🙂 Do you think we can incorporate it in some other way that makes more sense?

File: package.json
Comment (jonaslagoni) [reply package.json:25]: There is no such thing as `The AsyncAPI maintainers`, we have codeowners, and thats it 😄 So I just thought this would grasp everyone 🙂

File: CODEOWNERS
Comment (jonaslagoni) [reply CODEOWNERS]: Fixed

File: README.md
Comment (jonaslagoni) [reply README.md:124]: What if we added a statement about expectations from this library in the top? 🤔

**Comment by jonaslagoni** [comment]
@alequetzalli if you have time I would love for you to review this 🙂

File: README.md
Comment (magicmatatjahu) [reply README.md:124]: But that it's obvious to me that we have documentation for describing features/edge cases/usage etc and we have examples for show edge cases/usage API etc. That's what they are for :)

`<joke>` we are not journalists of some rag where they pay us for the obvious `</joke>` 😄 

We have on TOC `Documentation` and `Examples` nodes so for me it;s enough :)

File: package.json
Comment (magicmatatjahu) [reply package.json:25]: Ok then :)

File: README.md
Comment (jonaslagoni) [reply README.md:124]: Fair 😆

**Review by magicmatatjahu — APPROVED** [review]
LGTM, but Alejandra should look at this :)

File: README.md
Comment (alequetzalli) [inline README.md:129]:
```suggestion
We have gathered all the examples in a separate folder and they can be found under the [examples folder](./examples). 
```

**Review by alequetzalli — DISMISSED** [review]
LGTM, just had 1 suggestion ✨✨

File: README.md
Comment (magicmatatjahu) [inline README.md:14]: I know that it's only a badge, but we have also note in the Readme:

image

and then we describe (as comment to the badge) `The project has reached a stable, usable state and is being actively developed.`

`stable` isn't good term in this place 😅

**Review by magicmatatjahu — APPROVED** [review]
LGTM but I have only one comment, please refer to it before merge :)

File: README.md
Comment (magicmatatjahu) [inline README.md:17]: Duplicated `LICENSE`

File: README.md
Comment (jonaslagoni) [reply README.md:14]: That's honestly a good point, lets switch to `WIP` 🤔

File: README.md
Comment (jonaslagoni) [reply README.md]: Good catch 👁️, changed.

**Review by magicmatatjahu — APPROVED** [review]
LGTM

**Pull request merged**
